.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "concordium-contracts-common"]
+	path = concordium-contracts-common
+	url = git@github.com:Concordium/concordium-contracts-common.git
```

This path should be relative, 
```suggestion
	url = ../concordium-contracts-common.git
```